Frontend/ui-driver/src/Flow.purs

```diff
@@ -2565,7 +2565,9 @@ noInternetScreenFlow triggertype = do
                         false ->  baseAppFlow false Nothing
     TURN_ON_GPS -> if not internetCondition then noInternetScreenFlow "INTERNET_ACTION"
                     else do
-                      (DriverActiveInactiveResp resp) <- Remote.driverActiveInactiveBT "true" $ toUpper $ show Online
+                      when (isTokenValid (getValueToLocalStore REGISTERATION_TOKEN)) $ do 
+                        (DriverActiveInactiveResp resp) <- Remote.driverActiveInactiveBT "true" $ toUpper $ show Online
+                        pure unit 
```

Comment:
can you update it as `void $  Remote.driverActiveInactiveBT "true" $ toUpper $ show Online` no need of `pure unit`